ledger/participant-integration-api/src/main/scala/platform/index/IndexServiceImpl.scala

```diff
@@ -127,6 +140,13 @@ private[index] class IndexServiceImpl(
           to,
         )
         .map(_._2)
+
+      InstrumentedSource
```

Comment:
ideas: you could do chaining, so no additional identifier is needed. Or adding an implicit decorator to InstrumentedSource.

Reply:
I was experimenting with this in a previous PR and I prefer a fluent API style as well. But decided to not do it here since I would also like to unify the API with what we have in https://github.com/digital-asset/daml/blob/main/ledger/sandbox-on-x/src/main/scala/com/daml/ledger/sandbox/bridge/validate/ConflictCheckingLedgerBridge.scala#L45-L57